packages/ecs-agent/ecs-agent.spec

```diff
@@ -0,0 +1,102 @@
+%global goproject github.com/aws
+%global gorepo amazon-ecs-agent
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.41.0
+# git rev-parse --short=8
+%global gitrev 3776bee9
+
+%global source_date_epoch 1234567890
+%global tar_cf tar --sort=name --mtime="@%{source_date_epoch}" --owner=0 --group=0 --numeric-owner -cf
+
+Name: %{_cross_os}ecs-agent
+Version: %{gover}
+Release: 1%{?dist}
+Summary: Amazon Elastic Container Service agent
+License: Apache-2.0
+URL: https://%{goimport}
+Source0: https://%{goimport}/archive/v%{gover}.tar.gz
+Source1: ecs.service
+Source2: pause-config.json
+Source3: pause-manifest.json
+Source4: pause-repositories
+
+# Upstream: https://github.com/aws/amazon-ecs-agent/pull/2513
+# Upstream status: Merged
+Patch0001: 0001-engine-move-default-image-exclusions.patch
+
+# Bottlerocket-specific - filesystem location of the pause image
+Patch0002: 0002-bottlerocket-default-filesystem-locations.patch
+
+# Bottlerocket-specific - version data can be set with linker options
+Patch0003: 0003-bottlerocket-version-values-settable-with-linker.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+
+Requires: %{_cross_os}docker-engine
+# for sysctl
+Requires: %{_cross_os}procps
```

Comment:
We might not need this if we decide not to try to reset the `localnet` value in the service unit.

Reply:
It's still used for `ExecStartPre=/sbin/sysctl -w net.ipv4.conf.all.route_localnet=1`.  Would you prefer that we directly write to the `/proc/sys/net/ipv4/conf/all/route_localnet` file instead?

Reply:
We can use a sysctl.d setting like `release-sysctl.conf`, but included in the agent package.

Reply:
Done